irondb/api/data-deletion.md

```diff
@@ -655,69 +471,72 @@ DELETE
 #### Inputs
 
  * `uuid` : The UUID of the check to which the metric belongs.
- * `metric` : The name of the metric from which data is deleted.
- * `metric_pattern_including_wildcards` : A metric naming pattern string including wildcards.
+ * `metric` : The full canonical name of the metric from which data is deleted, including tags.
  * `query` : See [Tag Queries](/irondb/metric-names-tags-queries/#tag-queries) for more info on tag queries.
 
 #### Headers
 
-Used only with wildcards or tag query:
+Used with all deletes::
  * `x-snowth-account-id: <account_id>` (required)
    * `account_id` The account to be searched using the wildcard pattern
- * `x-snowth-advisory-limit: <integer>|none` (optional, defaults to no limit if not present)
+Used on with tag query delete:
+ * `x-snowth-advisory-limit: <integer>|none` (optional, defaults to 100,000 if not present)
    * `integer` A positive integer specifying the number of matching results to
-     delete. If the header is unset, or set to -1 or "none", the service will
-     not limit the result set.
+     delete. If the header is unset, the default of 100,000 will be used.
+Used only with canonical metric delete:
```

Comment:
This isn't used with all types of deletes?

Reply:
`full/canonical` is specifically for deleting a single metric. It matches a UUID and canonical metric name, which means that it is by definition only one metric.

Reply:
I misunderstood, not enough surrounding lines. I thought you were saying that the confirm-delete header was only for canonical, but it's whatever comes _after_ that "only with canonical" line. My mistake.

Reply:
Have changed to this:

```
Used with all deletes:
 * `x-snowth-account-id: <account_id>` (required)
   * `account_id` The account to be searched using the wildcard pattern
Used only with tag query delete:
 * `x-snowth-advisory-limit: <integer>|none` (optional, defaults to 100,000 if not present)
   * `integer` A positive integer specifying the number of matching results to
     delete. If the header is unset, the default of 100,000 will be used.
 * `x-snowth-confirm-delete: <0 or 1>` (optional, must be present and set to 1 to actually confirm and process the deletion)
```
I believe this to be correct.